SimulationOutputReport/tests/output_report_test.rb

```diff
@@ -29,6 +29,7 @@ class SimulationOutputReportTest < MiniTest::Test
     'Electricity: Mech Vent (MBtu)',
     'Electricity: Whole House Fan (MBtu)',
     'Electricity: Refrigerator (MBtu)',
+    'Electricity: Dehumidifier (MBtu)',
```

Comment:
`AnnualRows` and `TimeseriesColsComponentLoads` should have new "Component Load: Heating: Dehumidifier" and "Component Load: Cooling: Dehumidifier" rows too, no? Even if the file has no dehumidifier, they should be present but have zero values.

Reply:
Yeah I understood why it was expected here, but currently, dehumidifier was included in internal gain subcategory for component loads program. 
It is not hard to separate if it kept being confusing. 😛

Reply:
Ah, I see. We can keep it in internal gains for now until someone asks for it, that's fine.